security-ldap/.ignore

```diff
@@ -0,0 +1 @@
+Ignored for now due to https://github.com/oracle/graal/issues/3961
```

Comment:
What is this `.ignore` file?

Reply:
It was a way in Spring Native to prevent a smoke test to run as we know it wasn't working. Now that we have isolated task per test, this is no longer needed.

Reply:
Ah, thanks for clarification.